packages/accounts-base/accounts_client.js

```diff
@@ -1,51 +1,132 @@
-// @summary Constructor for AccountsClient instances. Available only on
-//          the client for now, though server code might also want to
-//          create a client connection to an accounts server.
-// @locus Client
-// @param options {Object} an object with fields:
-// - connection {Object} Optional DDP connection to reuse.
-// - ddpUrl {String} Optional URL for creating a new DDP connection.
-AccountsClient = function _AccountsClient(options) {
-  AccountsCommon.call(this, options);
-
-  this._loggingIn = false;
-  this._loggingInDeps = new Tracker.Dependency;
-
-  this._loginServicesHandle =
-    this.connection.subscribe("meteor.loginServiceConfiguration");
+/**
+ * @summary Constructor for the `Accounts` namespace on the client.
```

Comment:
Maybe let's call this "client object" instead of namespace? I feel like a namespace isn't something you can construct. Maybe this is too nitpicky.